bot/seasons/evergreen/8bitify.py

```diff
@@ -0,0 +1,54 @@
+from io import BytesIO
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+
+class EightBitify(commands.Cog):
+    """Make your avatar 8bit!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @staticmethod
+    def pixelate(image: Image):
```

Comment:
This method is missing a return type annotation

```suggestion
    def pixelate(image: Image) -> Image:
```